CPS-XXXX/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CPS: ?
+Title: Ouroboros not accounting for typical global network delays causes centralisation
+Status: Open
+Category: ?
```

Comment:
```suggestion
Category: Consensus
```
As I understand this is a recommendation for the `Consensus` protocol... please @bwbush correct me if I'm wrong about this (e.g. if in fact this is a challenge for the `Network` layer: cc @jpraynaud).